samcli/local/docker/container.py

```
@@ -219,6 +226,26 @@ def start(self, input_data=None):
         # Start the container
         real_container.start()
 
+    def get_logs_stream(self, follow=True):
+        """
+        Returns the container stdout and stderr streams as single iterator.
+        Parameters
+        ----------
+        follow: bool
+            Determines if we need to follow the previous log stream, or get the log stream from the beginning
+        Returns
+        -------
+        Iterator
+            an iterator of container stdout and stderr output strings.
+        """
+        if not self.is_created():
+            raise RuntimeError("Container does not exist. Cannot get logs for this container")
```

Should we raise an error here, or log an error message?